### DISCUSSION TITLE
Wrapper on iOS doesn't allow payment with Apple Pay through Stripe

### DISCUSSION THREAD

**Discussion by gianni-gulino-eleva** [DivanteLtd/vue-storefront#6713] [Q&A]
Hi everyone, I have a problem with Stripe integration on VSF.
I have integrated the payment via Stripe on a VSF project, the payment via Google Pay and Apple Pay works fine.

The problem arises when I go to generate the application for iOS via wrapper (https://github.com/DivanteLtd/vsf-mobile-wrapper-ios) using XCode: in this case, the button for payment with Apple Pay does not appear.

I assume the problem may be related to the wrapper running on a browser other than Safari.

Any ideas on how to fix this?

Thanks in advance

**Comment by Fifciu** [comment]
It might be related to the in-app payments policy of Apple. As far as I remember they didn't allow to use payment gateways not provided by them. Not sure if it's still the case but it might be a good first thing to check.
